src/rpc/util.h

```diff
@@ -16,4 +23,17 @@ CPubKey HexToPubKey(const std::string& hex_in);
 CPubKey AddrToPubKey(CKeyStore* const keystore, const std::string& addr_in);
 CScript CreateMultisigRedeemscript(const int required, const std::vector<CPubKey>& pubkeys);
 
+class DescribeAddressVisitor : public boost::static_visitor<UniValue>
```

Comment:
In https://github.com/bitcoin/bitcoin/pull/10583/commits/4dadd7c7ac0934447b8ca33942535cc8a22e78f5

Curious: if we're looking to eventually get away from using Boost, should we reserve continued/new usage of it for cases when the feature at hand would be hard to replicate on our own? Seems like each use of
```
boost::apply_visitor(DescribeAddressVisitor(), foo);
```
could be replaced with `DescribeAddressVisitor()(foo)`. 

I guess usages of `static_visitor` are relatively easy to unwind if we ever want to, and it may be worthwhile to remain consistent with our usage of it in the wallet code.

Feel free to ignore; not proposing we hold this changeset up on the basis of Boost usage, just curious.